src/schema/objects/extensions.yaml

```
@@ -140,6 +140,13 @@
 
     Used by KIT, Yokogawa, and Ricoh MEG systems.
     Successor to the `.sqd` extension for marker files.
+.ngff/:
```

I wonder if it should be `.zarr/`?!

If I get it right - [NGFF](https://ngff.openmicroscopy.org/) pretty much establishes a "schema" for (meta)data which can be then expressed in a container such as HDF5 (file) or ZARR (directory).  The prototype implementation is [said](https://ngff.openmicroscopy.org/latest/#ome-ngff) to be based on "OME-Zarr [ome-zarr-py](https://ngff.openmicroscopy.org/latest/#biblio-ome-zarr-py)" and [documentation uses .zarr/](https://ngff.openmicroscopy.org/latest/#image-layout) folders (not `.ngff/`!) since I guess it could as well be `.hdf5` files etc in other implementations. 

Hence, my concern that we might want to avoid adding a suffix which generally is not used?!

May be it should be `.ome.zarr/`?

---

Let's ask OME/NGFF gurus. @sbesson @joshmoore -- should we use `.ngff` or `.zarr` (or `.ome.zarr` as we have `.ome.tif`) extension for NGFF (ome-zarr) folders while introducing NGFF in BIDS?

---

👋🏽 Hi all. This is a good question that's still under discussion. The working consensus is that implementations SHOULD write filesets with the ending `.ome.zarr` but that they also SHOULD read filesets with the `.zarr` ending.

The reasoning for this is that an OME-NGFF dataset could be stored in a Zarr fileset along with data adhering to another specification. If OME-NGFF tried to "own" the file-ending, certain linkages of data would not be possible.

As far as I know, no implementations are currently reading or writing specifically with the `.ngff` file-ending.

---

fyi ping @LeeKamentsky

---

Thank you @joshmoore , so I think we should make it 

```suggestion
.ome.zarr/:
```

then to be consistent with other `.ome.` extensions